packages/fxa-profile-server/lib/img-workers.js

```diff
@@ -15,22 +15,32 @@ const WORKER_URL = config.get('worker.url');
 const ACCEPT_ENCODING_HEADER = 'identity';
 
 exports.upload = function upload(id, payload, headers) {
-  return new P(function (resolve, reject) {
+  return new P(function(resolve, reject) {
     var url = WORKER_URL + '/a/' + hex(id);
 
+    const header_excludes = config.get('worker.headers_exclude');
+    const worker_headers = Object.assign({}, headers);
+    if (header_excludes.length) {
+      // filter incoming headers
+      for (const exclude of header_excludes) {
+        delete worker_headers[exclude];
```

Comment:
I do like this approach of removing the unwanted headers.  However, we lose the case insensitivity comparisons from earlier; I think we should keep the case insensitivity.

Reply:
Dammit...

Reply:
done.